x/evidence/testutil/app.yaml

```diff
@@ -45,6 +50,10 @@ modules:
     config:
       "@type": cosmos.genutil.module.v1.Module
 
+  - name: mint
```

Comment:
how come is mint needed here? we should be able to do only staking and no mint and slashing?

Reply:
Well, thanks for pointing this! This actually fixes my bug.
I thought you needed mint if you add this to the auth account:

```
- account: mint
  permissions: [minter]
```